src/pages/developers/_components/Showcase.astro

```diff
@@ -0,0 +1,40 @@
+---
+import { getImage } from "astro:assets"
+import ThemeTwo from "../_assets/themes/theme-2.png"
+import ThemeSeven from "../_assets/themes/theme-7.png"
+import ThemeEight from "../_assets/themes/theme-8.png"
+import ThemeNine from "../_assets/themes/theme-9.png"
+
+const themeTwo = await getImage({ src: ThemeTwo, format: "webp", width: 1080 })
+const themeSeven = await getImage({ src: ThemeSeven, format: "webp", width: 1080 })
+const themeEight = await getImage({ src: ThemeEight, format: "webp", width: 1080 })
+const themeNine = await getImage({ src: ThemeNine, format: "webp", width: 1080 })
+---
+
+<section class="mt-10 md:mt-20 lg:mt-32">
+    <div class="w-full h-[20rem] md:h-[30rem] sm:h-[40rem] relative flex items-center justify-center overflow-hidden">
+
+        <div class="absolute z-50 w-full h-full bg-gradient-to-b from-transparent via-[#060913] via-70% to-95% to-[#060913]">
+        </div>
+
+        <div class="w-3/4 p-2 absolute top-6 z-40 bg-[#111521] border border-[#1E2231] rounded-2xl">
+            <img class="rounded-xl w-full object-cover object-top" src={themeTwo.src} alt="AstroWind Theme">
```

Comment:
Rather than using the low-level `getImage` API, you should refactor these to use Astro's [`<Image />` component](https://docs.astro.build/en/guides/images/#image--astroassets)

Reply:
I've updated this too, now using the Image component